README.md

```diff
@@ -1761,6 +1763,9 @@ IEEE Trans. Robot. (TRO), 2024. [Project page](https://github.com/tub-rip/cmax_s
 - <a name="Zhou21tro"></a>Zhou, Y., Gallego, G., Shen, S.,  
 *[ESVO: Event-based Stereo Visual Odometry](https://doi.org/10.1109/TRO.2021.3062252)*,  
 IEEE Trans. Robot. (TRO), 2021. [Project page](https://sites.google.com/view/esvo-project-page/home), [PDF](https://arxiv.org/pdf/2007.15548), [YouTube](https://youtu.be/3CPPs1gz04k), [Code](https://github.com/HKUST-Aerial-Robotics/ESVO.git).
+- <a name="niu25tro"></a>Niu, J., Zhong, S., Lu, X., Shen, S., Gallego, G. and Zhou, Y.,  
```

Comment:
This entry should be a bit further down, to keep some chronological order. Could you?

Reply:
@Ethan-Zhou